tests/test_utils.py

```diff
@@ -433,7 +458,7 @@ def test_type_str_none() -> None:
     [
         (Optional[int], "Optional[int]"),
         (Union[str, int, Color], "Union[str, int, Color]"),
-        (Optional[Union[int]], "Optional[int]"),
+        (Optional[int], "Optional[int]"),
```

Comment:
I believe the use of `Optional[Union[int]]` here was intentional. Note the `Optional[int]` case above.

Reply:
I'll push a commit to revert this.